qiita_pet/templates/analysis_selected.html

```diff
@@ -1,8 +1,15 @@
 {% extends sitebase.html %}
 {% block head %}
 <script type="text/javascript">
+var ws = new WebSocket('ws://' + window.location.host + '/analysis/selected/socket/');
+ws.onmessage = function(evt) { if(evt.data != 'true') {ws.close();} };
```

Comment:
Maybe? I may be being overly cautious, but at this point my thinking is better safe than sorry.